src/Business Foundation/App/Permissions/BusFoundObj.permissionset.al

```diff
@@ -0,0 +1,11 @@
+namespace Microsoft.Foundation;
+
+using Microsoft.Foundation.NoSeries;
+
+permissionset 1 "Bus. Found. - Obj."

```

Comment:
Will we be able to declare longer object names at some point in the future? 🤔

Reply:
For me this raises the question if camelCase/PascalCase would not be the better option here. 
If I counted correctly it would save 7 characters

Reply:
@christianbraeunlich You should ask that question in the forum, I can't give you an answer here in the PR. At least we cannot depend on that in this PR.
@StefanMaron Can you explain how camelCase/PascalCase would save 7 characters?..

Reply:
![image](https://github.com/microsoft/BCApps/assets/34504100/28869584-91c9-4f0b-842e-3f148fb509ec)
With PascalCase: BusFoundObj ;-)

Reply:
@AndreasMoth I already decided for my self a few years ago that I am not following the naming convention of objects anymore, and this was a very good example why.
Instead of `"Bus. Found. - Obj."` I would call it `BusFoundObj` or even choose less abbreviations since there are more characters available that way.

Reply:
I see, currently we are trying to give full names with spaces to everything, though I agree this one does seem a bit special. Will look more into what we can do, but I don't believe we will go down the route of Pascal casing. We may be able to do something using the new namespaces, though just "Objects" seem a bit too generic.